Sources/ProjectSpec/SpecParsingError.swift

```diff
@@ -5,6 +5,7 @@ public enum SpecParsingError: Error, CustomStringConvertible {
     case unknownTargetPlatform(String)
     case invalidDependency([String: Any])
     case unknownSourceBuildPhase(String)
+    case invalidSourceCopyFilesPhase
```

Comment:
This might get unwieldy if we add more build phase enum options. Can we just used the existing `unknownSourceBuildPhase` but renamed to `invalidSourceBuildPhase` along with the error. Then the string can be `copyFiles must be a dictionary`

Reply:
Yep. I’ll adjust.